omniscidb/L0Mgr/L0Mgr.h

```diff
@@ -68,6 +99,7 @@ class L0Device {
   std::shared_ptr<L0CommandQueue> command_queue_;
 
  public:
+  L0DataFetcher data_fetcher;
```

Comment:
Can the `data_fetcher_` be an implementation detail? Based on the class API I don't expect us ever to use it without a `device` object anyway.